cmd/provider/main.go

```diff
@@ -42,6 +43,7 @@ func main() {
 		timeout                = app.Flag("timeout", "Controls how long Ansible processes may run before they are killed.").Default("20m").Duration()
 		leaderElection         = app.Flag("leader-election", "Use leader election for the controller manager.").Short('l').Default("false").OverrideDefaultFromEnvar("LEADER_ELECTION").Bool()
 		maxReconcileRate       = app.Flag("max-reconcile-rate", "The maximum number of concurrent reconciliation operations.").Default("1").Int()
+		retryFailed            = app.Flag("retry-failed", "Whether to retry failed runs with ObserveAndDelete policy (with CheckWhenObserve, they are retried unconditionally).").Default("false").Bool()
```

Comment:
Idk if this flag is a good idea: retrying failed syncs, to me, is something that a k8s controller should always do, it shouldn't even be configurable. And it's unconditionally retried with CheckWhenObserve policy.
But there might be provider-ansible users who already rely on the existing no-retries behavior, which is the only reason I added the flag. LMK what you'd prefer

Reply:
:facepalm: yup forgot to consume it in `handleLastApplied`
Anyway, removed the flag